User journey

Valioesi opened the pull request:
Goto /unterschreiben page to see the first draft the signature list journey.
You can add url param step (downloadedList, printedList, signedList, sharedList, sentList) to finish each steps. 

This is just a draft. I don't want to create a pr yet, because some stuff is still missing. E.g. secondary ctas for some steps. And we are probably going to add the download of the list on the /unterschreiben page before we render the new component. @schaerta we probably need to talk again about how we are gonna do i exactly. We already spoke about it last week I think.

schaerta commented:
Could you increase the vertical spacing between cards a little bit? On my large screen I get distracted by the second card.

![image](https://user-images.githubusercontent.com/17318585/157441248-4582dcc4-5705-4757-93ae-215f4fd133b7.png)

Valioesi commented:
Yeah sure, but probably not tooo much, otherwise it might look weird on smaller screens.

Valioesi commented:
Alright, this is ready for review now. Added the link between downloading and "checkpoints" page. Goto /demokratie-fuer-alle to test. 
Also added missing secondary ctas and receiving a success param, so emails can link to an attribute on the page without setting it to done.

schaerta commented:
Und für die Demokratie-Initiative würde ich das Wording von "[Hilfe, ich habe keinen Drucker!]" einfach zu "Listen per Post bestellen" ändern. Sonst ist der Link auf innn.it doch etwas verwirrend.

schaerta commented:
Was auch noch fehlt, ist eine Erfolgsmeldung am Ende. Wir hatten ja noch nicht definiert, was als nächstes passieren soll, weil wir nicht wussten, ob wir die Listen-Eintragung nutzen können. Können wir erstmal einfach ein kleines Modal einfügen, in dem steht 

"Vielen Dank, dass du den Brief abgeschickt hast und der Initiative zum Erfolg verhilfst! 
Möchtest du noch mehr tun? Dann schließ dich doch einem unserer Sammelevents an! Schau mal auf der Karte vorbei um zu sehen, wo Events in deiner Nähe stattfinden."
("auf der Karte") wäre dann einfach der Link zurück zur Unterseite mit Sprungmarke.
